doc/manuals/admin/extra/ha.md

```diff
@@ -0,0 +1,441 @@
+          "name" : "mongo2:27017",
+          "health" : 1,
+          "state" : 1,
+          "stateStr" : "PRIMARY",
+          "uptime" : 3967,
+          "optime" : {
+            "ts" : Timestamp(1486052399, 1),
+            "t" : NumberLong(5)
+          },
+          "optimeDate" : ISODate("2017-02-02T16:19:59Z"),
+          "lastHeartbeat" : ISODate("2017-02-02T17:25:55.623Z"),
+          "lastHeartbeatRecv" : ISODate("2017-02-02T17:25:55.623Z"),
+          "pingMs" : NumberLong(0),
+          "electionTime" : Timestamp(1486052398, 1),
+          "electionDate" : ISODate("2017-02-02T16:19:58Z"),
+          "configVersion" : 3
+        }
+      ],
+      "ok" : 1
+    }
+    ```
+
+### Context Broker configuration
+
+For each server:
+
+1. Configure the Firewall to block access from outside the local network except 
+   for SSH and on local network for 1026
+
+    ```bash
+    $ nano /etc/sysconfig/iptables
+    
+    # Firewall configuration written by system-config-firewall
+    # Manual customization of this file is not recommended.
+    *filter
+    :INPUT ACCEPT [0:0]
+    :FORWARD ACCEPT [0:0]
+    :OUTPUT ACCEPT [0:0]
+    -A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+    -A INPUT -p icmp -j ACCEPT
+    -A INPUT -i lo -j ACCEPT
+    -A INPUT -m state --state NEW -m tcp -p tcp --dport 1026 -s 10.0.64.0/25 -j ACCEPT
+    -A INPUT -m state --state NEW -m tcp -p tcp --dport 22 -j ACCEPT
+    -A INPUT -j REJECT --reject-with icmp-host-prohibited
+    -A FORWARD -j REJECT --reject-with icmp-host-prohibited
+    COMMIT
+    
+    
+    $ iptables-restore < /etc/sysconfig/iptables
+    ```
+
+1. Configure the hosts file to include local network hostname resolution
+
+    ```bash
+    $ nano /etc/hosts
+
+    10.0.64.32   mongo1
+    10.0.64.33   mongo2
+    10.0.64.34   mongo3
+
+    10.0.64.32   mongo1.lab.fiware.org
+    10.0.64.33   mongo2.lab.fiware.org
+    10.0.64.34   mongo3.lab.fiware.org
+    10.0.64.35   contextbroker1.lab.fiware.org
+    10.0.64.36   contextbroker2.lab.fiware.org
+    10.0.64.37   contextbroker3.lab.fiware.org
+    10.0.64.38   proxy1.lab.fiware.org
+    10.0.64.39   proxy2.lab.fiware.org
+    10.0.64.40   proxy3.lab.fiware.org
+    ```
+
+1. Install the Context Broker with the following commands
+
+    ```bash
+    $ nano /etc/yum.repos.d/fiware.repo
+
+    [fiware]
+    name=Fiware Repository
+    baseurl=http://repositories.lab.fiware.org/repo/rpm/$releasever
+    gpgcheck=0
+    enabled=1
+
+    $ yum install contextBroker
+    ```
+
+1. Configure the Context Broker by listing all the mongodb hosts and the name
+   of the replica set
+
+    ```bash
+    $ nano /etc/sysconfig/contextBroker
+
+    BROKER_DATABASE_HOST=mongo1,mongo2,mongo3
+    BROKER_DATABASE_RPLSET=contextBroker
+    ```
+
+1. Start the Context Broker
+
+    ```bash
+    $ /etc/init.d/contextBroker start
+    ```
+
+1. Test the Context Broker
+
+    ```bash
+    $ curl localhost:1026/version -s -S | python -mjson.tool
```

Comment:
The pretty printer `python -mjson.tool` is not needed in the case of GET /version operation. The response is already pretty-printed (this is an exception of the general rule of CB responses ;)